[docs] fix sentence of `<svelte:element>` (#7484)
sveltejs/svelte: site/content/docs/02-template-syntax.md

---

> The `<svelte:element>` element lets you render an element of a dynamically specified type. This is useful for example when rich text content from a CMS. Any properties and event listeners present will be applied to the element.

Reviewer: https://github.com/sveltejs/svelte/pull/7483#discussion_r857120120

While we're here, might as well fix it in this same PR.
```suggestion
The `<svelte:element>` element lets you render an element of a dynamically specified type. This is useful for example when displaying rich text content from a CMS. Any properties and event listeners present will be applied to the element.
```

Or w/e else fits here

Author: I agree.

The only supported binding is `bind:this`, since the element type specific bindings that Svelte does at build time (e.g. `bind:value` for input elements) does not work with a dynamic tag type.

> If `this` has a nullish value, element and its children are not displayed.

Reviewer 2: ```suggestion
If `this` has a nullish value, the element and its children will not be rendered.
```

Author: Thank you! I should go to English school! 😅
